umm-spec-lib/resources/json-schemas/service/umm/v1.0/umm-s-json-schema.json

```diff
@@ -1,139 +1,83 @@
 {
-  "$schema": "http://json-schema.org/draft-05/schema#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
why are we reverting to an older version of the JSON schema spec?

Reply:
> why are we reverting to an older version of the JSON schema spec?

No idea, just copy paste what Simon puts in the DRAFT repo. I think the original "05" is wrong as collections and variables are all at 04. So this change is good.